includes/class-wc-google-analytics-js.php

```diff
@@ -243,7 +243,7 @@ public static function load_analytics_universal( $logged_in ) {
 		$ga_snippet_head = "(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
 		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
 		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-		})(window,document,'script', $src" . self::tracker_var() . "');";
+		})(window,document,'script', '$src','" . self::tracker_var() . "');";
```

Comment:
When embedding PHP within string sometimes it gets lost when "eyeing it".  So the only suggestion would be to enclose it with braces like `'{$src}'` So that it stands out more.